docs/code_samples/default_async.txt

```diff
@@ -0,0 +1,60 @@
+// works for NODE > v10
+const axios = require("axios");
+const fs = require("fs");
+const FormData = require("form-data");
+const {setTimeout} = require("node:timers/promises");
+
+const apiKey = "my-api-key";
+const account = "my-account";
+const endpoint = "my-endpoint";
+const version = "my-version";
+
+const urlRoot = `https://api.mindee.net/v1/products/${account}/${endpoint}/v${version}`;
+
+async function makeRequest(url, method, data = null) {
+  const config = {
+    method: method,
+    url: url,
+    headers: { Authorization: `Token ${apiKey}` },
+    data
+  }
+  try {
+    const response = await axios(config);
+    return response.data;
+  } catch (err) {
+    console.error(err);
+    process.exit(1);
+  }
+}
+
+async function enqueueRequest(filePath) {
+  const data = new FormData();
+  data.append("document", fs.createReadStream(filePath));
+  return await makeRequest(`${urlRoot}/predict_async`, "POST", data);
+}
+
+async function pollRequest(jobId) {
+  return await makeRequest(`${urlRoot}/documents/queue/${jobId}`, "GET");
+}
+
+async function main(filePath) {
+  const enqueueResponse = await enqueueRequest(filePath);
+  await setTimeout(6000);
```

Comment:
```suggestion
  await setTimeout(4000);
```
Blank sample takes considerably less time. Might want to decrease this to avoid unnecessary long CI checks.

Reply:
What's important is that the sample works for non-blank files ...

Reply:
:crying_cat_face: